lib/sgtn-client/api/source.rb

```diff
@@ -68,4 +38,4 @@ def self.getBundle(component, locale)
 
   end
 
-end
\ No newline at end of file
+end
```

Comment:
Add bundle['messages'] = bundle.delete locale after line 32
```suggestion
end
```